src/main/kotlin/io/snyk/plugin/Utils.kt

```diff
@@ -430,10 +430,30 @@ fun String.toVirtualFile(): VirtualFile {
     return if (!this.startsWith("file://")) {
         StandardFileSystems.local().refreshAndFindFileByPath(this) ?: throw FileNotFoundException(this)
     } else {
-        VirtualFileManager.getInstance().refreshAndFindFileByUrl(this) ?: throw FileNotFoundException(this)
+        VirtualFileManager.getInstance().refreshAndFindFileByUrl(this.toVirtualFileURL()) ?: throw FileNotFoundException(this)
     }
 }
 
+// add a slash when on windows
+fun VirtualFile.toLanguageServerURL(): String {
+    if (this.urlContainsDriveLetter()) {
+        return this.url.replace("file://","file:///")
+    }
+    return this.url
+}
+
+// remove first "/" if on windows
+fun String.toVirtualFileURL(): String {
+    if (this.isWindowsURI() && this.startsWith("file:///") && this.length > 10 && this.substring(9,10) == ":") {
+        return this.replaceFirst("/","")
+    }
+    return this
+}
+
+fun String.isWindowsURI() = SystemUtils.IS_OS_WINDOWS && this.startsWith("file://")
+
+fun VirtualFile.urlContainsDriveLetter() = this.url.isWindowsURI() && this.url.length > 9 && this.url.substring(8,9) == ":"
```

Comment:
nit: It might be nicer to do this using a regex. I really had to think about what this function was doing, although the tests did help. Just a personal preference though

Reply:
Hm... I'm so-so. For the last part it would definitely be easier with a regexp. At least if you understand/know regular expressions ;).

Reply:
Would be something like this: `file:\/\/(\/)?[a-zA-Z]:.*$`. Do you think that's easier to understand?

Reply:
![image](https://github.com/snyk/snyk-intellij-plugin/assets/20150761/9a614d32-f61c-4e68-90b1-eedcc57618d4)

Reply:
We agreed on a call to keep it as it was! Turns out the regex is no easier 😆